aci/resource_aci_rtctrlattrp.go

```diff
@@ -199,6 +509,60 @@ func resourceAciActionRuleProfileRead(ctx context.Context, d *schema.ResourceDat
 	return nil
 }
 
+func resourceAciRtctrlSetCommRead(ctx context.Context, dn string, d *schema.ResourceData, m interface{}) diag.Diagnostics {
```

Comment:
Not sure those read functions are necessary (see above comments on the double reads)